bloecke/pep-block_dotted.tex

```diff
@@ -0,0 +1,172 @@
+% Measurements:
+% Margins:
+%   top:    1.15cm
+%   bottom: 1.15cm
+%   left:   1.05cm
+%   right:  1.05cm
+% Logo:
+%   height: 2cm
+% Grid:
+%   height: 28.0cm
+%   width:  19.5cm
+% Watermark:
+%   diameter:   15.0cm
+
+% DINA4 Size: 21 cm x 29,7 cm
+% 21cm - (37 * 0.5 cm) = 21cm - 17.5cm = 3.5cm
+% left and right margins: 3.5cm * 0.5 = 1.75cm
+
+\RequirePackage{luatex85}
```

Comment:
not needed anymore for recent TeX Live versions

Reply:
removed